setup.py

```diff
@@ -24,6 +24,7 @@ def read(fname):
     author_email="devs.aspire@gmail.com",
     install_requires=[
         "click",
+        "colorednoise",
```

Comment:
I'm dubious about adding more dependencies here, especially for something that we can do ourselves (using the unfortunately named `aspire.image.xform.NoiseAdder`). It might be better to just strip out that noise-generating function and use that for these examples.

Reply:
Sure, I can try using the noise adder. If it works that probably would be better. I'll try it this morning.

Understood re depends.  When I found the package, I looked at the code and it didn't concern me (likely it has worked and will continue forward indefinitely).  As you note below, it doesn't do exactly what we want anyway (though I thought it did enough to flex the code a little).

Reply:
I was able to use noise adder to add white noise.  It isn't clear to me how to do anything more than that.

Reply:
You give it a noise filter that defines the desired PSD of the noise. Something like
```
f = FunctionFilter(lambda x: np.exp(-x ** 2 / (2 * 0.3 ** 2))
```
should give interesting results. Another option is
```
f = FunctionFilter(lambda x: 1 / (1 + x ** 2))
```

Reply:
Cool, thanks a lot, big help. I was looking at the PowerFilter etc and it didn't make sense to me.

Reply:
Again, I think this is a docs issue. Discoverability is a big problem.